src/reconcile.py

```diff
@@ -678,8 +678,9 @@ def index_page(
         The action to take for the index page.
     """
     table_contents = "\n".join(table_row.to_markdown(discourse.host) for table_row in table_rows)
+    # Strip any whitespace around file contents to avoid building up more and more whitespace
```

Comment:
Remove "more and more" just 'avoid building up whitespace' should be good

Reply:
Thanks, fixed